### DISCUSSION TITLE
Add support for mass transfers

### DISCUSSION THREAD

**Discussion by ajjswift** [pelican-dev/panel#20] [Ideas]
Pelican should add reliable support for the mass transfer of servers between nodes, instead of having to transfer them one-by-one.

**Comment by lancepioch** [comment]
What kind of scale are we talking about here, how many servers are you typically transferring and how often?

**Reply by ajjswift** [reply]
Entire nodes, not hugely often.

Useful for decommissioning old nodes to upgrade to newer hardware. Currently it's a pain having to transfer one-by-one.

**Reply by starrider346** [reply]
Node transfer seems like a really useful thing.

**Reply by HerrSammyDE** [reply]
> What kind of scale are we talking about here, how many servers are you typically transferring and how often?

@lancepioch 

from an entrepreneurial perspective:

We had transferred a few hundred servers (just under 1k) from Datacenter A to Datacenter B. We had to write something ourselves so that we didn't have to transfer each server independently.

So it would be very cool!
